SourceCpp/Diffusion.cpp

```diff
@@ -136,6 +136,10 @@ PeleC::getMOLSrcTerm(
         }
         continue;
       }
+      // Note on typ: if interior cells (vbox) are all covered, no need to
```

Comment:
So why not making `typ` above be on `cbox` and not `vbox` that way it is not redefined. Also it would match F90 implementation better.

Reply:
That's the way I had it originally. The downside is that if you have a FAB that is all covered cells on the interior but has cut cells in the ghost region you would end up going through the whole getMOLSrcTerm calculation for no reason. I don't think this would actually hurt anything, but it would be slightly inefficient in the cases where it comes up.

In the end I don't think it makes much difference, so I'm happy to switch it back to the more simple way if you prefer it.

Reply:
got it! I was being dumb